byoeb-v1/byoeb/byoeb/scripts/ragas_model_comparison_v2.py

```
@@ -0,0 +1,1049 @@
+        from ragas.llms import llm_factory
+        from ragas.embeddings import embedding_factory
+        from datasets import Dataset
+        from ragas.run_config import RunConfig
+    except ImportError as e:
+        print(f"Missing dependency: {e}. Run: pip install ragas datasets")
+        return {}
+
+    print(f"\n--- RAGAS evaluation for {model} (chunk={_RAGAS_CHUNK_SIZE}, max_workers=4) ---")
+
+    from openai import OpenAI as _OpenAI, AsyncOpenAI as _AsyncOpenAI
+    _oai_sync  = _OpenAI(api_key=openai_key, organization=None, http_client=httpx.Client(verify=False))
```

Ruff (0.15.13)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)